Rakefile

```
@@ -13,7 +13,13 @@ require "rake/testtask"
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList["test/**/*_test.rb"]
+  t.test_files = FileList["test/**/*_test.rb"] - ["test/ruby_lsp_rails/server_test.rb"]
 end
 
-task default: [:"db:setup", :test]
+Rake::TestTask.new("test:server") do |t|
+  t.libs << "test"
+  t.libs << "lib"
+  t.test_files = ["test/ruby_lsp_rails/server_test.rb"]
+end
+
+task default: [:"db:setup", :test, "test:server"]
```

Did you test if adding a `present?` in the add-on code breaks when running `bundle exec rake`?

I _think_ because `test:server` is last, then the related test helper only gets required at the end, but I'm not 100% sure.

```suggestion
task default: ["db:setup", :test, "test:server"]
```